src/memex/views.py

```diff
@@ -82,7 +99,41 @@ def api_config(**settings):
         request_method = ('DELETE', 'GET', 'HEAD', 'POST', 'PUT',)
     settings['request_method'] = request_method + ('OPTIONS',)
 
-    return view_config(**settings)
+    if link_name:
+        link = {'name': link_name,
+                'method': initial_request_method,
+                'route_name': settings.get('route_name'),
+                'description': description,
+                }
+
+        registry = config.registry
+        if not hasattr(registry, 'api_links'):
+            registry.api_links = []
+        registry.api_links.append(link)
+
+    config.add_view(view=view, **settings)
```

Comment:
`config.add_view` doesn't return a value. I generally prefer to avoid mocking but it turned out to be quite easy providing `add_api_view` took `config` as the first parameter.